app/AppComponents.scala

```diff
@@ -29,8 +29,8 @@ trait AWSComponent { self: BuiltInComponents =>
 
 trait ControllersComponent {
   self: BuiltInComponents with AWSComponent =>
-  def messagesApi: MessagesApi = new DefaultMessagesApi(environment, configuration, new DefaultLangs(configuration))
-  def appController = new Application(dynamoClient, talksTableName)
+  val messagesApi: MessagesApi = new DefaultMessagesApi(environment, configuration, new DefaultLangs(configuration))
```

Comment:
Yeah @marialivia16 and I had a problem and concluded these should b`def`s of `lazy val`s.